src/project.rs

```diff
@@ -142,7 +142,15 @@ impl FuzzProject {
             .arg(&build.triple);
         // we default to release mode unless debug mode is explicitly requested
         if !build.dev {
-            cmd.args(["--release", "--config", "profile.release.debug=true"]);
+            cmd.args([
+                "--release",
+                "--config",
+                "profile.release.debug=true",
+                "--config",
+                "profile.release.lto=false",
+            ]);
+        } else {
+            cmd.args(["--config", "profile.dev.lto=false"]);
```

Comment:
I think this is a comment on the first commit, at this commit the two config options are slightly different since they contain the profile name `profile.release.lto=false` vs `profile.dev.lto=false` which I think makes it hard to directly refactor as you suggest.

But this actually changes how you suggest in later commits as part of making the profile support more generic and we end up with:
```rust
        cmd.args([
            &format!("--profile={profile}"),
            "--config",
            &format!("profile.{profile}.debug=true"),
            "--config",
            &format!("profile.{profile}.lto=false"),
        ]);
```